modules/db/src/test/scala/gem/dao/ObservationDaoSpec.scala

```diff
@@ -64,14 +90,39 @@ class ObservationDaoSpec extends PropSpec with PropertyChecks with DaoTest {
     }
   }
 
-  property("ObservationDao should select target-only observations") {
-    val oid = Observation.Id(pid, Index.One)
+  property("ObservationDao should fail fetchStatic if missing") {
```

Comment:
What's `property`?

Reply:
Something from `scalatest` that "register[s] a property-based test with the specified name". I'm not a big fan of any of the test DSLs.  There are so many different ways to do the same thing and I never know which one to use.  I always just end up copying some existing test that seems similar and doing whatever it did.